src/tests/request_values_tests.rs

```diff
@@ -17,6 +17,21 @@ use uuid::uuid;
 
 #[napi]
 pub fn tests_from_value_get_type(test: String) -> ComplexType {
+    if test.as_str() == "Tuple" {
+        return ComplexType::two_support(
+            CqlType::Tuple,
+            Some(ComplexType::simple_type(CqlType::Text)),
+            Some(ComplexType::two_support(
+                CqlType::Tuple,
+                Some(ComplexType::simple_type(CqlType::Int)),
+                Some(ComplexType::two_support(
+                    CqlType::Tuple,
+                    Some(ComplexType::simple_type(CqlType::Empty)),
+                    None,
+                )),
+            )),
+        );
+    }
```

Comment:
:recycle: Can we make it another case of the `match` below? We can use `return` in its branch, so that we don't have to conform to other branches wrt return type.